File: functions/financial_calc_update.py
# Kaleb Carrillo, Financial Calculator Update python
def info(cost, income, type):
    percent = (cost/income)*100
    print(f"your {type} is ${cost:.2f}, Which is, {percent}, of your income. \n")


# print statment that welcomes user tells what the program does
print("Welcomes user this is a fanacial calculator for your needs.")
# ask what their income is (varible an input)
income = float(input("What is your income?\n"))
# ask what their rent is (varible an input)
rent = float(input("What is your rent?\n"))
# ask what their utlitites is (varible an input)
utlities = float(input("What is your utlitites cost?\n"))
# ask what their groceries is (varible an input)
groceries = float(input("What is groceries cost?\n"))
# ask what their transportation is (varible an input)
transportation = float(input("what is your transportation cost?\n"))
# calculate savings as 10% of income (income*.1) (varible)
savings = (income*.1)
saving = (savings/income)*100
# calculate sav ing as income-savings-rent-utilites-groceries-transportaion (varible)
spending = income-savings-rent-utlities-groceries-transportation
spendings =(spending/income)*100

info(rent, income, "rent")
info(utlities, income, "utities")
info(groceries, income, "groceries")
info(transportation, income, "transportation")
info(saving, income, "saving")
info(spending, income, "spending")

File: functions/financial_calc_update.c
// Kaleb Carrillo, Financial Calculator Update C
#include <stdio.h>
#include <math.h>

char name [1000];
float user_input(char*info){
    float value;
    printf("%s", info);
    scanf("%f", &value);
    return value;
}
void print_result(char*item_name, float num, float income){
    float percentage = (num/income)*100;
    printf("%s, your monthly %s is $%.2f which is %.2f%% of your income!\n", name, item_name, num, percentage);
}
int main(void){
    printf("Welcome user to my Financial calculator for your needs!\n");
    float income, rent, utilities, groceries, transportation, savings, spendings;

    income = user_input("What is your income?: \n");
    rent = user_input("What is your rent?: \n");
    utilities = user_input("How much do your utilities cost?: \n");
    groceries = user_input("How much do your groceries cost?: \n");
    transportation = user_input("How much does transportation cost?: \n");
    
    savings = income*0.1;
    spendings = income-savings-rent-utilities-groceries-transportation;

    print_result("rent", rent, income);
    print_result("utilities", utilities, income);
    print_result("groceries", groceries, income);
    print_result("transportation", transportation, income);
    print_result("savings", savings, income);
    print_result("spendings", spendings, income);
    
    return 0;
}